IntegrationTestProject/ProjectUnderTest/ProjectUnderTest.sln

```diff
@@ -0,0 +1,27 @@
+﻿
```

Comment:
Please move the folders/files out of the repository root to some appropriate under *Tests*.